images/flaky-test-reporter/Makefile

```diff
@@ -0,0 +1,32 @@
+
+# Copyright 2019 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+IMG = gcr.io/knative-tests/test-infra/flaky-test-reporter
+TAG := $(shell date +v%Y%m%d)-$(shell git describe --always --dirty --match '^$$')
+
+all: build
+
+build:
+	CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build ../../tools/flaky-test-reporter
```

Comment:
To avoid compatibility issues, you should build the tool from inside the container. Please see the prow-tests image Dockerfile for guidance.

Reply:
updated with building in docker